### PR TITLE
Dbutson/dashboards

### PR DESCRIPTION
**Pull request by butson**
remove redundant dashboards and fixed ops view.

### CODE REVIEW

File: conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json [hunk 1]
```diff
@@ -1000,159 +1045,26 @@
...
               ],
               "type": "fill"
             }
           ],
-          "hide": false,
-          "measurement": "Commits",
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT sum(\"rate\") AS \"TPS\", count(\"rate\") AS \"# TEs\" FROM \"Commits\" WHERE (\"nodetype\" = 'Transaction')  AND \"db\" =~ /^$database$/ AND \"host\" =~ /^$host$/ AND $timeFilter GROUP BY time($__interval) fill(none)",
+          "query": "SELECT sum(\"rate\") AS \"TPS\", count(\"rate\") AS \"# TEs\" FROM (SELECT mean(\"rate\") AS rate FROM \"Commits\" WHERE (\"nodetype\" = 'Transaction' AND \"db\" =~ /^$database$/ AND \"host\" =~ /^$host$/) AND $timeFilter GROUP BY time($__interval), \"host\" , \"db\", \"$pid\" fill(null)) WHERE $timeFilter GROUP BY time($__interval)",
```
Comment (sivanov-nuodb) [inline conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json:1064]: I assume that this is the relevant change and everything else is automatically changed by Grafana once the dashboard is exported?

**Review by sivanov-nuodb — APPROVED** [review]
Thanks for the change!

File: conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json (hunk 1 above)
Comment (butson) [reply conf/provisioning/dashboards/nuodb/nuodb-ops-overview.json:1064]: yes,  there were a few other's removed that were disabled.   The issue with dashboards is that the version is constantly changing.   An older grafana can load a newer dashboard,  however, some fields might be missing, additional fields will be discarded on load.

**Review by vegichan — APPROVED** [review]
looks good

**Pull request merged**